TypeSystem/flow-analysis/reachability_A22_t05.dart

```diff
@@ -13,7 +13,9 @@
 ///   - Let `after(N) = after(E2)`.
 ///
 /// @description Checks that if the static type of the expression of the form
-/// `E1.m1(E2)` static type of `E1`is not `Never` then `after(N) = after(E2)`.
+/// `E1.m1(E2)` static type of `E1`is not `Never` then `after(N) = after(E2)`,
+/// which is tested by detecting that `i = 42` is considered to be guaranteed
+/// to have been executed when `i;` is executed.
```

Comment:
This one needs an adjustment. Perhaps it is intended to say something like the following?:
```suggestion
/// @description Checks that for an expression of the form `E1.m1(E2)`, if the
/// static type of `E1`is not `Never` then `after(N) = after(E2)`. This is
/// tested by detecting that `i = 42` is considered to be guaranteed to have
/// been executed when `i;` is executed.
```